### DISCUSSION TITLE
bright theme the dark is not readable

### DISCUSSION THREAD

**Discussion by reset12** [alicevision/Meshroom#2501] [General]
Hello,
can you implement a bright theme, i cant read anything. It is only usable at night.

**Comment by robekras** [comment]
I had the same problem and I was searching very long to find a solution via search engines...
Searching within the meshroom source code has been successful...
There is a shortcut to toggle between dark mode and light mode: Ctrl + Shift + P.

It would be very useful to document such features in within the manual.

**Reply by reset12** [reply]
Thank you for your research.
